.github/ISSUE_TEMPLATE/1_bug_report.yml

```diff
@@ -0,0 +1,46 @@
+name: 🐛 Report a bug
+description: '___'
```

Comment:
If it works:

```suggestion
description: ___
```

Reply:
It works, 
and it might appear ok in the issue template chooser interface:
![2021-07-06_154519](https://user-images.githubusercontent.com/723651/124602422-804af980-de71-11eb-9de0-7ea4580ba806.jpg)
but showing `___. If this doesn’t look right, choose a different type` in the form itself, looks strange :
![2021-07-06_154540](https://user-images.githubusercontent.com/723651/124602444-87720780-de71-11eb-9a69-b0ff96200d9f.jpg)

Reply:
I committed your suggestion, but I still think we should some text, not just `___`

Reply:
“Report a bug” already says it all. Anything else is just redundant.

Reply:
Yes, of course. 
It's just that `___` seems strange to me, as if 3 characters that could not be displayed properly.

&nbsp;
And, according to the [documentation](https://gh-community.github.io/issue-template-feedback/structured/#:~:text=description%20(String)%3A%20A%20description%20of%20this%20template%27s%20intended%20use.%20This%20will%20be%20shown%20in%20the%20issue%20template%20chooser%20interface.), `display` is a required field (String), not optional,
and unfortunately it can't get an empty value, 
i.e. `description: "   "` or `description:  ` both result in the following when commiting:

```
 There is a problem with this template

About can't be blank.
```